### PR TITLE
New recipe for wfdb

### PR DESCRIPTION
**Pull request by Gijom**
wfdb is a toolbox to read and write WFDB files

### CODE REVIEW

File: recipes/wfdb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/wfdb/meta.yaml]: The `--single-version-externally-managed` flag to `setup.py` means that you only need packages in the list `build` requirements that are either imported by `setup.py` or are in `setup_requires`. I think you only need `python` and `setuptools` here.

File: recipes/wfdb/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
```
Comment (synapticarbors) [inline recipes/wfdb/meta.yaml]: Consider adding `noarch: python`

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

Comment (Gijom) [reply recipes/wfdb/meta.yaml]: Indeed that seems the toolbox is writtent in pure python.

File: recipes/wfdb/meta.yaml (hunk 1 above)
Comment (Gijom) [reply recipes/wfdb/meta.yaml]: Most build dependencies were removed, I only kept python and setuptool.

**Comment by jakirkham** [comment]
Thanks @Gijom! Also thanks @synapticarbors for reviewing. :)

**Pull request merged**